frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseRenameAction.java

```diff
@@ -76,13 +77,14 @@ public void actionPerformed(AnActionEvent anActionEvent) {
   private void rename(GitRepository gitRepository, BaseEnhancedGraphTable graphTable, IManagedBranchSnapshot branch,
       BranchLayout branchLayout) {
     val project = gitRepository.getProject();
+    val remotes = new ArrayList<>(gitRepository.getRemotes());
 
     val gitNewBranchDialog = new GitNewBranchDialog(project, Collections.singletonList(gitRepository),
         getString("action.GitMachete.BaseRenameAction.description").fmt(branch.getName()),
         branch.getName(),
         /* showCheckOutOption */ false,
         /* showResetOption */ false,
-        /* showSetTrackingOption */ false,
+        /* showSetTrackingOption */ branch.getRemoteTrackingBranch() != null,
```

Comment:
Huh would it be easily possible to rename this option from _Set tracking branch_ (which is unclear) to _Rename tracking branch_?

Reply:
I am  gonna to copy and revamp GitNewBranchDialog to achieve that :trollface: